app/views/home_page/_header.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="hero-wrapper">
+  <header>
+    <div class="logo">
```

Comment:
This is really very minor but a more suitable class here would be `logo-container`